data/meta/Body.lua

```diff
@@ -0,0 +1,144 @@
+-- Copyright © 2008-2022 Pioneer Developers. See AUTHORS.txt for details
+-- Licensed under the terms of the GPL v3. See licenses/GPL-3.txt
+
+-- This file implements type information about C++ classes for Lua static analysis
+
+---@meta
+
+---
+---@class Body
+---
+---@field label string
+---@field seed integer
+---
+--- The <SystemPath> that points to this body.
+--- Only available for non-dynamic bodies.
+---@field path? SystemPath
+---
+--- The type of the body, as a <Constants.BodyType> constant.
+---@field type? BodyType
+---
+--- The supertype of the body, as a <Constants.BodySuperType> constant
+---@field superType? BodySuperType
+---
+--- The non-dynamic body attached to the frame this dynamic body is in.
+--- Only available for dynamic bodies.
+---@field frameBody? Body
+---
+--- Whether the frame this body is in is a rotating frame.
+--- Only available for dynamic bodies.
+---@field frameRotating? boolean
+---
+
+---@class Body
```

Comment:
`[Lua Diagnostics. duplicate-doc-class] [W] Duplicate defined class.`

Reply:
Based on conversation in IRC, this appears to be caused by an old version of lua-language-server, as this diagnostic string is no longer present in the language server sources.